src/service/performance-impl.js

```diff
@@ -223,6 +263,30 @@ export class Performance {
       entryTypesToObserve.push('firstInput');
     }
 
+    if (this.win.PerformanceLayoutJank) {
+      // Programmatically read once as currently PerformanceObserver does not
+      // report past entries as of Chrome 61.
+      // https://bugs.chromium.org/p/chromium/issues/detail?id=725567
+      this.win.performance.getEntriesByType('layoutJank').forEach(processEntry);
+      entryTypesToObserve.push('layoutJank');
+
+      // Register a handler to record the layout jank metric when the page
+      // enters the hidden lifecycle state.
+      // @see https://developers.google.com/web/updates/2018/07/page-lifecycle-api
+      this.win.addEventListener(
+          'visibilitychange',
+          this.onVisibilityChange_,
+          {capture: true}
+      );
```

Comment:
Based on notes about capture usage in the [Page Lifecycle API docs](https://developers.google.com/web/updates/2018/07/page-lifecycle-api#observing-state-changes) we should be able to use a common listener in the bubble phase rather than capture.

Done in 363f278